packages/cli/api-importers/openapi/openapi-ir-parser/src/index.ts

```diff
@@ -1,2 +1,5 @@
 export { type ParseOpenAPIOptions } from "./options";
-export { parse, type Spec } from "./parse";
+export { parse, type Document } from "./parse";
+export { InMemoryOpenAPILoader } from "./openapi/InMemoryOpenAPILoader";
```

Comment:
i could see this being moved to the `browser-workspace` that you end up creating

Reply:
Great call - I have that tee'd up in the next PR 👍